FSCalendar/FSCalendarHeaderView.m

```diff
@@ -290,6 +290,11 @@ - (void)prepareLayout
     
 }
 
+- (BOOL)shouldInvalidateLayoutForBoundsChange:(CGRect)newBounds
+{
+    return self.collectionView ? !CGRectEqualToRect(self.collectionView.bounds, newBounds) : false;
```

Comment:
`false` should be `NO` in ObjC.

I am unsure, but I think self.collectionView if should be more explicit (self.collectionView != nil)

Reply:
Fixed